app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -35,6 +36,22 @@ def tenant=(name)
       attrs[:tenant] = name
     end
 
+    def allow_ext_net?
+      !(attrs[:allow_ext_net] == 0)
+    end
+
+    def allow_ext_net
+      attrs[:allow_ext_net]
+    end
+
+    def allow_ext_net=(enabled)
+      if ['true', true, '1', 1].include?(enabled)
```

Comment:
@imriz That will set it to `true` or `false`, which I think you don't want since you're validating for it to be 0 or 1. I don't know if the OpenStack API accepts both formats or just 1 and 0.